### PR TITLE
Add `dactyl_manuform_carbonfet` board

### PR DESCRIPTION
**Pull request by dunk2k**
Addition of the [carbonfet](https://github.com/carbonfet) thumb cluster variation of the dactyl_manuform to KMK

### CODE REVIEW

File: boards/dactyl_manuform_carbonfet/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+The [Dactyl ManuForm Carbonfet](https://github.com/carbonfet/dactyl-manuform) is a handwired, split bodied, tented, curved key-well keyboard.  
+Forked from the [Dactyl ManuForm Mini](/boards/dactyl_manuform_mini), the Dactyl ManuForm Carbonfet appends the 5 key thumb cluster of the former with an additional key into a 2 row by 3 column arrangement.
+
+![White](https://i.imgur.com/0ugz1C9.jpg)  
+*Dactyl Manuform Carbonfet 5x6 variant*
+
+## Variants
+
+Dactyl Manuform Carbonfet's are built in variations that cater for different row and column counts.
```
Comment (xs5871) [inline boards/dactyl_manuform_carbonfet/README.md:11]:
```suggestion
Dactyl Manuform Carbonfets are built in variations that cater for different row and column counts.  
```

File: boards/dactyl_manuform_carbonfet/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+![White](https://i.imgur.com/0ugz1C9.jpg)  
+*Dactyl Manuform Carbonfet 5x6 variant*
+
+## Variants
+
+Dactyl Manuform Carbonfet's are built in variations that cater for different row and column counts.  
+Variants are denoted as `RowCount`*x*`ColumnCount` and share the common configuration of:
+- The finger key-well bottom row has 2 keys; 1 each in ring and middle columns.
+    - Exception to this rule is the `5x7` variant that has two additional keys in this row.
```
Comment (xs5871) [inline boards/dactyl_manuform_carbonfet/README.md:14]:
```suggestion
   Exception to this rule is the `5x7` variant that has two additional keys in this row.
```
One element is not a list.

File: boards/dactyl_manuform_carbonfet/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+| 4 | Three rows, typically for alphabet and some punctuation characters, with two keys in bottom row of finger key-well section |
+| 5 | As *4 rows* with number row above |
+| 6 | As *5 rows* with function row above |
+
+### Column
+| Count | Description |
+| --- | --- |
+| 5 | A column for each finger with additional index finger column |  
+| 6 | As *5 columns* with additional pinky finger column |
+| 7 | As *6 columns* with either an additional index finger column (`5x7`) or additional pinky column (`6x7`) |
```
Comment (xs5871) [inline boards/dactyl_manuform_carbonfet/README.md:29]: Please align the column separators. (pad fields with spaces)

**Review by xs5871 — CHANGES_REQUESTED** [review]

File: boards/dactyl_manuform_carbonfet/README.md (hunk 3 above)
Comment (dunk2k) [reply boards/dactyl_manuform_carbonfet/README.md]: amended as of commit 082ba79

**Review by xs5871 — APPROVED** [review]

**Comment by dunk2k** [comment]
@xs5871 thanks 😊

**Pull request merged**